circle.yml

```diff
@@ -126,4 +124,8 @@ test:
       unicode
       unicode/utf16
       unicode/utf8
+    - >
+      gopherjs test --short --minify --run='^Test'
+      net/http/cookiejar
+      reflect
```

Comment:
I believe that's the only failure in the reflect package.  I was essentially being lazy about creating yet another special case to run the other examples, but I can do that.